### ISSUE TITLE
Create blog.webmaker.org/facesofmozilla

### ISSUE THREAD

**Issue by KevZawacki** [MozillaFoundation/plan#585]
Plan: Create a space for our MozFest profiles (https://github.com/MozillaFoundation/plan/issues/566) on blog.webmaker.org.

Profiles will be written and published during the month of October. But due to their volume, we don't want them to clutter up the main feed. Ideally, they'd be hidden from the landing page and accessible via a URL like blog.webmaker.org/facesofmozilla

@adamlofting Ideas on how we can do this in Wordpress?

cc @benrito

**Comment by adamlofting** [comment]
> @adamlofting Ideas on how we can do this in Wordpress?

It depends on how we've built our templates and manage our menus etc, which would need checking with whoever built them. I'm guessing we did this in hoouse. 

But one option is to publish the profiles as 'Pages' and not include them in the navigation. 

Looking at the live site, we don't appear to have any navigation menu, so it might be possible to just add pages without setting up anything new.
